src/domain/GetPaymentNotificationMetadataRecord.ts

```diff
@@ -44,7 +44,7 @@ const makePaymentNotificationAttachmentDownloadMetadataResponse =
     contentType: paymentNotificationAttachment.contentType,
     contentLength: 0,
     sha256: paymentNotificationAttachment.digests.sha256,
-    url: `${env.downloadDocumentURL.href}/${env.sampleStaticPdfFileName}`,
+    url: `${env.downloadDocumentURL.href}/${env.sampleStaticPdfFileName}?correlation-id=${env.iunGenerator()}`,
```

Comment:
This structure (`NotificationAttachmentDownloadMetadataResponse`) is the same as the one used in `GetNotificationDocumentMetadataRecord` file. Should we unify them? WDYT? Moreover, all three URLs are constructed in the same way

Reply:
Yeah, I agree to unify them! Thanks for the catch.

I'll push some updates with a refactor for this.

Reply:
I've made some changes! Let me know WDYT 🍺